.circleci/config.yml

```diff
@@ -12,16 +12,52 @@ version: 2.1
 orbs:
   win: circleci/windows@2.2.0
 
+_refs:
+  defaults: &defaults
+    working_directory: ~/cli
+    docker:
+      - image: node:12
+  ssh-config: &ssh-config
+    fingerprints:
+      - '3b:68:69:7d:19:a4:50:69:ad:b8:db:57:f9:9c:f0:3f'
+  restore_cache: &restore_cache
+    keys:
+      - dependency-cache-v2-{{ checksum "package.json" }}
+      # if cache for exact version of 'package.json' is not present, then load the most recent cache
+      - dependency-cache-v2-
+  install: &install
+    name: Install dependencies
+    command: |
+      npm --version
+      node --version
+      npm install
+  gh-config: &gh-config
+    name: Configuring GitHub
+    command: |
+      git config credential.helper 'cache --timeout=120'
+      git config user.email "$GH_EMAIL"
+      git config user.name "Release Bot"
+  gus-create-change-mgmt: &gus-create-change-mgmt
+    name: 'Gus create change case management'
+    command: |
+      npm install -g @salesforce/change-case-management
+      export GUS_BUILD="$(node -pe "require('./packages/salesforcedx-vscode/package.json').version")"
+      echo "Gus build is ${GUS_BUILD}"
+      sfchangecase create -i 500B0000005YGshIAG --release offcore.tooling.${GUS_BUILD} --location ${REPO_LOCATION}
```

Comment:
Is it ok to have a GUS Id checked in ?

Reply:
I don't think it's a problem since we're already posting links to work items in some of the issues but I'll move this to an environment variable.